tox.ini

```diff
@@ -27,8 +27,8 @@ deps=
     {[base]deps}
     pyside2_515: pyside2
     pyqt5_515: pyqt5
-    pyqt6_62: pyqt6~=6.2.0
-    pyqt6_62: PyQt6-Qt6~=6.2.0
+    pyqt6_62: pyqt6==6.2.0
+    pyqt6_62: PyQt6-Qt6==6.2.0
```

Comment:
Again, why is there the `==` and not `~=` here?

Reply:
This ensures that the same version is drawn in for both